data/json/npcs/talk_tags.json

```diff
@@ -629,14 +739,17 @@
     "text": [
       "<swear!> Fire in the hole!",
       "<swear!> Get cover!",
-      "Marines!  We are <swear> leaving!",
       "Hit the <swear> dirt!",
       "This <swear> shit is gonna blow!",
       "I'm standing way too close to this <swear> firecracker.",
       "I <really> need to get some distance.",
       "I <really> need to get some <swear> distance.",
       "<swear!> I'm getting my ass out of here!",
-      "Fire in the <swear> hole, motherfuckers!"
+      "Fire in the <swear> hole, motherfuckers!",
+      "Heads up, <name_b>.",
+      "Making some noise!",
+      "Watch out for shrapnel!",
+      "Watch out, I'm gonna make it rain men!"
```

Comment:
You could be throwing a grenade at a bear or a turret. This doesn't always make sense, because of context.